pegasus/emails/dashboard.html

```diff
@@ -0,0 +1,7 @@
+---
+from: '"Code.org" <noreply@code.org>'
+subject: "<%= subject %>"
+---
+<%= body %>
+
+<img src="<%= tracking_pixel %>"/>
```

Comment:
though we might actually need a "transactional" flag so that unsubscribe is ignored for these kinds of messages - we only want unsubscribe to apply to marketing emails.